jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/SchemaStore.java

```diff
@@ -18,26 +18,28 @@
 
 import static org.apache.commons.lang3.StringUtils.*;
 
+import com.fasterxml.jackson.databind.JsonNode;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.util.HashMap;
 import java.util.Map;
 
-import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
Out of curiosity: did some setting change or it is due to switch from Eclipse to IDEA that order of import rows is sometimes "jumping".
Maybe it would be possible to share idea Code Style (Settings -> Editor -> Code Style -> Java | Json) scheme to be used similarly to [jsonschema2pojo-eclipse-formatter.xml](https://github.com/joelittlejohn/jsonschema2pojo/blob/master/jsonschema2pojo-eclipse-formatter.xml) and [jsonschema2pojo-eclipse-preferences.epf](https://github.com/joelittlejohn/jsonschema2pojo/blob/master/jsonschema2pojo-eclipse-preferences.epf) ?

Reply:
Yes, I'm still getting used to the options in IDEA. I think I've set things up now so that it will be the same as Eclipse.

I'm a reluctant IDEA user tbh :smile:

I've added a jsonschema2pojo-intellij-formatter.xml now too.